app/src/components/market/sections/market_buy/market_buy.tsx

```diff
@@ -251,7 +253,12 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
       )
       setDisplayAmountToFund(new BigNumber('0'))
       setStatus(Status.Ready)
-      setMessage(`Successfully bought ${sharesAmount} '${balances[outcomeIndex].outcomeName}' shares.`)
+      {
+        Number(sharesAmount) < 0.01
+          ? setMessage(`Successfully bought <${0.01} '${balances[outcomeIndex].outcomeName}' shares.`)
+          : setMessage(`Successfully bought ${sharesAmount} '${balances[outcomeIndex].outcomeName}' shares.`)
+      }
+
```

Comment:
I think we shouldn't repeat whole message and setMessage here but just make one function and number in it and make it format...